extensions/jackson/deployment/src/main/java/io/quarkus/jackson/deployment/JacksonProcessor.java

```diff
@@ -111,6 +112,8 @@ public class JacksonProcessor {
     @BuildStep
     void unremovable(Capabilities capabilities, BuildProducer<UnremovableBeanBuildItem> producer,
             BuildProducer<AdditionalBeanBuildItem> additionalProducer) {
+        additionalProducer.produce(AdditionalBeanBuildItem.unremovableOf(ConfigurationCustomizer.class));
```

Comment:
Is this the correct way of doing it? Or should I use the `producer` instead of the `additionalProducer`?

Reply:
This should work just fine :)